app/services/idv/inherited_proofing/service_provider_services.rb

```diff
@@ -0,0 +1,26 @@
+module Idv
+  module InheritedProofing
+    module ServiceProviderServices
+      include Idv::InheritedProofing::ServiceProviders
+
+      def inherited_proofing_service_for(service_provider, service_provider_data:)
+        inherited_proofing_service_class_for(service_provider).new(service_provider_data)
+      end
+
+      def inherited_proofing_service_class_for(service_provider)
+        unless IdentityConfig.store.inherited_proofing_enabled
+          raise 'Inherited Proofing is not enabled'
+        end
+
+        if service_provider == VA
```

Comment:
ditto about inlining
```suggestion
        if service_provider == :va
```

Reply:
Done.